init.lua

```diff
@@ -26,13 +34,18 @@ serverFiles = nil
 -- Connect to the WiFi access point. Once the device is connected,
 -- you may start the HTTP server.
 tmr.alarm(0, 3000, 1, function()
-   if wifi.sta.getip() == nil then
+
+   if wifi.sta.getip() == nil and joincounter < 5 then
       print("Connecting to AP...")
-   else
+      joincounter = joincounter +1
+   else 
       tmr.stop(0)
-      print('IP: ',wifi.sta.getip())
+      -- print('IP: ',wifi.sta.getip())
       -- Uncomment to automatically start the server in port 80
-      -- dofile("httpserver.lc")(80)
+      joincounter = nil
+      collectgarbage()
+      dofile("httpserver.lc")(80)

```

Comment:
I'd like to leave this commented out in the repository, to match the comment but also because it's also easier for development.